SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -52,14 +52,18 @@ UnitTest {
 
 	// run all UnitTest subclasses
 	*runAll {
-		^this.forkIfNeeded {
-			this.reset;
-			this.allSubclasses.do { |testClass|
-				testClass.run(false, false);
-				0.1.wait;
-			};
-			this.report
-		}
+		if(this.class.name == 'Meta_UnitTest', {
```

Comment:
you could just test for `if(this.class == UnitTest, {`

Reply:
@dyfer when you change this, you could also change the formatting of the `if` to use trailing closure syntax, so it's more in line with the rest of the codebase (`if(x) { y } { z }` rather than `if(x, {y}, {z})`